suricata.yaml.in

```diff
@@ -660,6 +660,56 @@ af-packet:
     #use-mmap: no
     #tpacket-v3: yes
 
+dpdk:
+  eal-params:
+    proc-type: primary
+    a: 0000:3b:00.0
+    l: 0-1
+
+  # DPDK capture support
+  # Currently queues should be assigned to cores in ratio in 1:1 ratio
+  interfaces:
+    - interface: 0000:3b:00.0 # PCIe address of the NIC port
+      threads: auto # or number of threads - auto takes all lcores, no thread entry takes all cores
+      rx-queues: auto # or number of queues - auto assigns each core 1 queue
+      tx-queues: auto # or number of queues - auto assigns each core 1 queue
+      promisc: true
+      multicast: true # enables also detection on multicast packets
+      checksum-checks: true # if Suricata should validate checksums
+      checksum-checks-offload: true # if possible offload checksum validation to the NIC
+      mtu: 3000 # Set MTU of the device in bytes # default 1526
```

Comment:
MTU is highly dependent on 
1. NIC PMD support
2. Mempool elements vs perforamnce
3. mbuf changing is enabled.

Suricata expects one continuous virtual address for processing layer of ethernt, IP, udp/tcp and others. Hence enabling MTU is valid as long the mempool  size is set to be higher for each elements. Please do all necessary validation once parse YAML for DPDK is done. Hence re-work

Reply:
please check what happens when Jumbo frames are received as mutli-segments in DPDK. Then check suricata will it work.